README.md

```diff
@@ -232,6 +232,7 @@ By default, `ccd-docker` runs the most commonly used backend and frontend projec
 Optional compose files will allow other projects to be enabled on demand using the `enable` and `disable` commands.
 
 * To enable **document-management-store-app**
+  * add the following entry to your `/etc/hosts` file: `127.0.0.1   dm-store`
```

Comment:
As discussed, let's add 1 line to explain that this is for when you are running `XUI` outside `ia-docker` but `dm-store` is running within `ia-docker`